packages/react-cookie/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.0.30",
   "license": "MIT",
   "description": "Cookies in React for the server and client",
-  "main": "dist/src/index.js",
-  "types": "dist/src/index.d.ts",
+  "main": "index.js",
+  "types": "index.d.ts",
```

Comment:
For packages to get the full benefits of the different build outputs, you'll eventually need to also add `esnext`/ `module` entries to package.json (and, if you're interested, the future-looking `exports` map). Example: https://github.com/lemonmade/quilt/blob/main/packages/quilt/package.json#L10-L32